src/components/TabBar.tsx

```diff
@@ -7,13 +7,16 @@ interface TabBarProps<T> {
   isSelected: (item: T) => boolean
   isHighlighted?: (item: T) => boolean
   tabTextBuilder?: (item: T) => string
+  focus: boolean
 }
 
 export default function TabBar<T>(props: TabBarProps<T>) {
   const determineTabColor = (item: T) => {
     if (props.isSelected(item)) return "tab-selected"
     if (props.isHighlighted && props.isHighlighted(item))
       return "tab-highlighted"
+    if (props.focus && !props.isSelected(item))
```

Comment:
second condition here is unnecessary, if it were true the function would have already returned by this point.

also, can we rename `props.focus` to something more descriptive, like `props.greyOutBackgrounded` or something lol

Reply:
Updated